.github/workflows/github_build_action.yml

```diff
@@ -0,0 +1,53 @@
+name: github/actions
+on: push
```

Comment:
This could be more specific e.g. 
on:
  pull_request:
    branches: master
    
 This wel also rerun the build if another push is made to the PR

Reply:
thx, will update